WordPress/Classes/ViewRelated/NUX/SigninLinkMailViewController.swift

```diff
@@ -34,6 +34,8 @@ class SigninLinkMailViewController: NUXAbstractViewController {
         }
 
         localizeControls()
+        
+        self.sourceTag = SupportSourceTag.wpComLogin
```

Comment:
Would it make sense to specify this as part of the wpcom magic link flow vs the default wpcom screenname / password flow?